tslint.json

```diff
@@ -81,7 +81,7 @@
     // Disallows importing the whole RxJS library. Submodules can be still imported.
     "import-blacklist": [true, "rxjs", "rxjs/operators"],
     // Avoids inconsistent linebreak styles in source files. Forces developers to use LF linebreaks.
-    "linebreak-style": [true, "LF"],
+    "linebreak-style": [false, "LF"],
```

Comment:
were the changes to this file made intentionally? I don't think we want to change these settings

Reply:
Absoluto noy. I'm uses to switch those to false just to run tslint locally, and this time I forgot to switch them back after tests.